examples/tutorials/advanced/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1112 @@
+{
```

Comment:
<p>Add a dot (.) at the end of " about <a href="https://blog.openmined.org/federated-learning-types/" target="_blank">different kinds of Federated Learning</a>)"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done.</p>